Html the reckoning

Apogate opened the pull request:
**ADDED IN PULL:**

~~Although it keeps the original, disgusting table usage for thread~~ This pull revamps html layout for every page. This switches Saguaro from the old futallaby table based page system to a css+div system with class name for individual elements.

**Supported stylesheets so far are Sagurichan, Saguaba and Tomorrow.**

_Burichan, kusaba, and monotone are no longer supported. This update breaks those stylesheets._

Futallaby-esque and unused elements have been removed from the page.

**To-do**

<hr>
- [x] Index/thread rewriting
- [x] Mobile display
  - [x] Images expand properly
  - [ ] Mobile theme compatible with current supported theme **As of 10-14, only Sagurichan is supported** 
- [x] _head.php_ rewritten
  - [ ] Mobile support
- [x] _foot.php_ rewritten
  - [ ] Mobile support

Apogate committed:
Removing redundant/unused futallaby CSS, new HTML

Apogate committed:
This didnt make it in i guess

test

Apogate committed:
Disable broken image search by default

Apogate committed:
Re-add name to sagurichan. whoops

Apogate committed:
Correct subject coloring in sagurichan. whoops x2 combo

RePod commented:
Interesting movement of `Post->formatOP()`.

Apogate commented:
@RePod  it makes more sense since that's the way the pages are rendered anyway.

Apogate committed:
Shuffling elements around, containers for everyone!

Adds a couple divs for later manipulation.

Apogate committed:
Tidy up index html page. Everything is nested neatly now

Apogate committed:
Whoops x3 wombo combo

Apogate committed:
More containers.

Apogate commented:
my pulll comment is not coherent, i'll rewrite that eventually

RePod commented:
Ensure changes to index generation in **_core/log/log.php** are also in **_core/index/index.php** for when the Index class is eventually used proper.

Apogate committed:
It's time to go mobile

This adds full mobile support for most major browsers. Currently only
available in Sagurichan theme. Mobileshitters enjoy.

Apogate committed:
Add NSFW_BOARD config attribute

Apogate commented:
Some bugs so far:
 - OP comment isn't displayed in thread reply mode

- Image hover interferes with mobile image expansion:
    - Disable by default? Or detect if mobile user, and disable? Probably the first

Apogate commented:
Post numbers don't display in mobile view.

Also, untested with spoilers, IDs enabled or long post #s

Apogate committed:
Fixes comments not being inline with image.

Apogate committed:
Finalize mobile support.

For now, finalize all mobile support. Tested and resolved bugs. HTML has
been mostly converted and this should be the last major change to page
structure.
